modules/services/cloud-connector/variables.tf

```diff
@@ -43,12 +43,6 @@ variable "sns_topic_arn" {
 # optionals - with default
 #---------------------------------
 
-variable "sysdig_secure_endpoint" {
-  type        = string
-  default     = "https://secure.sysdig.com"
-  description = "Sysdig Secure API endpoint"
-}
-
```

Comment:
humm, any reason to remove this? :thinking:

Reply:
Whoops, removed it from the wrong file... I'll add it back in